--- a/video/out/hwdec/hwdec_aimagereader.c
+++ b/video/out/hwdec/hwdec_aimagereader.c
@@ -152,7 +152,7 @@
     // dummy dimensions, AImageReader only transports hardware buffers
     media_status_t ret = p->AImageReader_newWithUsage(16, 16,
         AIMAGE_FORMAT_PRIVATE, AHARDWAREBUFFER_USAGE_GPU_SAMPLED_IMAGE,
-        5, &p->reader);
+        9, &p->reader);
     if (ret != AMEDIA_OK) {
         MP_ERR(hw, "newWithUsage failed: %d\n", ret);
         return -1;
